llms/hf_llm/requirements.txt

```diff
@@ -1,3 +1,4 @@
 mlx>=0.0.7
 numpy
 transformers
+protobuf
```

Comment:
This is to solve the tokenizer error for the Yi model. Otherwise, we have to disable use_fast in the tokenizer.

Reply:
Should we enable `use_fast` in the tokenizer? It is not enabled right now.

Reply:
The use_fast is enabled by default. -> https://github.com/huggingface/transformers/blob/main/src/transformers/models/auto/tokenization_auto.py#L668